matter_server/common/custom_clusters.py

```diff
@@ -55,6 +62,17 @@ def __init_subclass__(cls: ClusterAttributeDescriptor, *args, **kwargs) -> None:
         ALL_CUSTOM_ATTRIBUTES[cls.cluster_id][cls.attribute_id] = cls
 
 
+def should_poll_eve_energy(node_data: MatterNodeData) -> bool:
+    """Check if the (Eve Energy) custom attribute should be polled for state changes."""
+    if node_data.attributes.get("0/40/2") != 4874:
+        # Some implementation (such as MatterBridge) use the
+        # Eve cluster to send the power measurements. Filter that out.
+        return False
+    # if the ElectricalPowerMeasurement cluster is NOT present,
+    # we should poll the custom Eve cluster attribute(s).
+    return node_data.attributes.get("2/144/65531") is None
```

Comment:
In what order do things happen after update? Do we get an attribute updates first before we check what needs polled? :thinking:

Reply:
I think we have to assume that this will be at random. So worst case scenario we still poll the custom value after a device gets updated. That is no big deal. At some point when the matter server is restarted (e.g. after an update) it will stop polling. Sure we could try to handle that (edge) case but to what purpose other than writing even more complicated code.